src/autowiring/DispatchQueue.cpp

```diff
@@ -80,12 +82,12 @@ void DispatchQueue::Abort(void) {
 void DispatchQueue::WaitForEvent(void) {
   std::unique_lock<std::mutex> lk(m_dispatchLock);
   if (m_aborted)
-    throw dispatch_aborted_exception();
+    throw dispatch_aborted_exception("Dispatch queue was aborted while waiting for an event");
```

Comment:
if you can differentiate exceptions with different error strings, do so.  Makes it a lot easier to track exceptions to a line number.